# TrackEntityMsg 
string label 
bool success
float32[] roi
integer lostframes

